File: tema-02/areaRectangulo.php
<!DOCTYPE html>
<!--
Escribe un programa que calcule el área de un rectángulo.
-->
<html>
    <head>
        <title>Area Rectangulo</title>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
    </head>
    <body>
        <div>
            <h3>Area Rectangulo</h3>
            <?php 
                $altura = $_GET["altura"];
                $base = $_GET["base"];
                $area = $altura * $base;
                
                echo 'Resultado ', $area, ' metros cuadrados';
            ?>
        </div>
    </body>
</html>

File: capitolul3/src/capitolul3/AreaRectangulo.java
/*
 Escribe un programa que calcule el área de un rectángulo.
*/
package capitolul3;

import java.util.Scanner;

/**
 *
 * @author costy
 */
public class AreaRectangulo {
    public static void main(String[] args) {
        Scanner t = new Scanner(System.in);
        
        int base;
        int altura;
        
        System.out.print("Introduce la base del rectangulo:");
        base = t.nextInt();
        System.out.print("Introduce la altura del rectangulo:");
        altura = t.nextInt();
        
        double area = (double)(base * altura);
        System.out.println("El area del rectangulo es de " + area + " cm cuadrados");
    }
}
